### PR TITLE
feat(soffit-order): new admin api

### PR DESCRIPTION
**Pull request by AliMD**

### CODE REVIEW

File: uniquely/com-api/src/route/patch-price-list.ts [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 import {nanoServer} from '../lib/server.js';
 import {storageClient} from '../lib/storage.js';
+import {validateUserAuth} from '../lib/validate-user-auth.js';
 
 import type {ProductPrice} from '@alwatr/type/customer-order-management.js';
 
 nanoServer.route('PATCH', '/price-list/', async (connection) => {
   logger.logMethod?.('patch-price-list');
-  connection.requireToken(config.nanoServer.adminToken);
+
```
Comment (njfamirm) [inline uniquely/com-api/src/route/patch-price-list.ts:10]: all user access them

File: uniquely/com-api/src/route/patch-product-list.ts [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 import {storageClient} from '../lib/storage.js';
+import {validateUserAuth} from '../lib/validate-user-auth.js';
 
 import type {Product} from '@alwatr/type/customer-order-management.js';
 
 nanoServer.route('PATCH', '/product-list/', async (connection) => {
   logger.logMethod?.('patch-product-list');
-  connection.requireToken(config.nanoServer.adminToken);
+
+  await validateUserAuth(connection.getUserAuth());
```
Comment (njfamirm) [inline uniquely/com-api/src/route/patch-product-list.ts:11]: all user access this

File: uniquely/com-api/src/route/patch-user.ts [hunk 3]
```diff
@@ -29,40 +27,22 @@ nanoServer.route('PATCH', '/user-list/', async (connection) => {
...
 
-  return {
-    ok: true,
-    data: await userStorage.set(user),
-  };
-});
+  const privateUserOrderListStorageName = config.privateStorage.userOrderList.replace('${userId}', user.id);
+  await storageClient.touch(privateUserOrderListStorageName);
 
-const saveSeparateUserProfile = (user: ComUser): void => {
```
Comment (njfamirm) [inline uniquely/com-api/src/route/patch-user.ts:40]: ?

File: uniquely/com-api/src/route/user-list.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import {nanoServer} from '../lib/server.js';
+import {storageClient} from '../lib/storage.js';
+import {validateUserAuth} from '../lib/validate-user-auth.js';
+
+nanoServer.route('GET', '/user-list/', async (connection) => {
+  logger.logMethod?.('get-user-list');
+
+  await validateUserAuth(connection.getUserAuth(), '*');
+
+  const userList = await storageClient.getStorage(config.privateStorage.userList);
```
Comment (njfamirm) [inline uniquely/com-api/src/route/user-list.ts:11]: why define in objec?

Comment (AliMD) [reply uniquely/com-api/src/route/user-list.ts]: هویجوری 🥕

**Pull request merged**